package.json

```diff
@@ -66,12 +66,10 @@
   },
   "lint-staged": {
     "**/*.ts?(x)": [
-      "npm run lint:fix",
-      "git add"
+      "npm run lint:fix"
     ],
     "**/*.{scss,css,html}": [
-      "prettier --print-width 120 --write",
-      "git add"
+      "prettier --print-width 120 --write"
```

Comment:
I removed the `git add` command based on the message in the console when comitting:

![git-add_message](https://user-images.githubusercontent.com/5565841/114243410-7e4cc980-995a-11eb-9299-941f7737799b.PNG)